pages/pension/application/527EZ.md

```diff
@@ -1,19 +0,0 @@
----
-title: Apply For Pension Benefits
-heading: Apply for pension benefits
-display_title: Apply now
-description: Submit your VA pension application (VA Form 21P-527EZ) online now. Get step-by-step instructions, and consider signing in to save your in-progress application if you need to come back later to finish filling it out.
-entryname: pensions
-layout: page-react.html
-spoke: Get benefits
-collection: pension
-order: 3
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/">Home</a></li>
-    <li><a href="/pension/">Pension benefits</a></li>
-    <li><a aria-current="page" href="/pension/how-to-apply/">Apply for Veterans Pension</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-75740cb768bec60be16d906b8118a271R5. Breadcrumbs look good.